pkg/controller/useraccount/useraccount_controller_test.go

```diff
@@ -118,6 +118,7 @@ func TestReconcileOK(t *testing.T) {
 				toolchainv1alpha1.Condition{
 					Type:   toolchainv1alpha1.UserAccountReady,
 					Status: corev1.ConditionFalse,
+					Reason: "Provisioning",
```

Comment:
I guess, we can use const introduced at https://github.com/codeready-toolchain/member-operator/pull/30/commits/90db32a057df8635ec0f412c5aaeccb843a6bb74#diff-25293d17133b9d89712a15c8e56795a3R35

Reply:
done https://github.com/codeready-toolchain/member-operator/pull/30/commits/4d9628b13cc8cff29b8e5f1928135fa306069c28